src/clj/rems/email/template.clj

```diff
@@ -150,7 +150,9 @@
                         :t.email.reviewed/message))
 
 (defmethod event-to-emails :application.event/remarked [event application]
-  (emails-to-recipients (handlers application)
+  (emails-to-recipients (concat (handlers application)
+                                (when (:application/public event)
+                                  (application-util/applicant-and-members application)))
```

Comment:
I think we sort of agreed on the channel that perhaps this sort of communication would go only to the applicant and the most important one i.e. approval or reject to all?

Reply:
Yeah, I used `applicant-and-members` here for consistency with the other emails. Should we remove it for every email?

Reply:
After discussion on Flowdock, removed it from only this case.